packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableWithdrawalRow.tsx

```diff
@@ -439,20 +431,20 @@ export function TransactionsTableWithdrawalRow({
       </td>
 
       <td
-        className={twMerge('w-1/5 px-3 py-3', isCustomAddressTx ? 'pb-10' : '')}
+        className={twMerge('w-1/5 px-3 py-3', customAddressTx ? 'pb-10' : '')}
       >
         <WithdrawalRowTxID tx={tx} />
       </td>
 
       <td
         className={twMerge(
           'relative w-1/5 py-3 pl-3 pr-6 text-right',
-          isCustomAddressTx ? 'pb-10' : ''
+          customAddressTx ? 'pb-10' : ''
```

Comment:
after repeating so many times i feel like it should be `customAddressTxPaddingBottom` lol

Reply:
fair point